pkg/webhook/clusterpropagationpolicy/mutating.go

```diff
@@ -51,10 +51,6 @@ func (a *MutatingAdmission) Handle(_ context.Context, req admission.Request) adm
 		return admission.Errored(http.StatusBadRequest, fmt.Errorf("ClusterPropagationPolicy's name should be no more than %d characters", validation.LabelValueMaxLength))
 	}
 
-	if helper.ContainsServiceImport(policy.Spec.ResourceSelectors) {
```

Comment:
This modification may not be appropriate to include in the current pull request. It may be necessary to make modifications in the next sync ServiceImport pull request.